packages/lib-react-native-jsi/README.md

```diff
@@ -50,246 +50,14 @@ Then, rename `AppDelegate.m` to `AppDelegate.mm` in order to compile this file a
 
 `AppDelegate.mm`
```

Comment:
This also should be removed if your removed the code bellow ;)

Reply:
Thanks! Fixed.